docs/api/api/api.txt

```diff
@@ -1944,7 +1944,7 @@ GET /staging/<staging_workflow_project>/staging_projects?requests=1&status=1&his
   Get the overall state of all staging projects belonging to a staging workflow project.
   Extra information can be requested by adding any combination of these parameters in the URL: requests, status and history.
 
-  - If requests is present, the output includes the staged requests.
+  - If requests is present, the output includes the staged, untracked and obsole requests as well as missing reviews.
```

Comment:
`obsole` looks like a typo. Did you mean `obsolete`?

Reply:
wow, 3 pairs of eyes to overlook it. Good catch

Reply:
I'll send a PR to fix this.